content/blog/introducing-visual-open-authoring.md

```diff
@@ -8,40 +8,34 @@ consumes: null
+The ability for a website to run statically by default, and then "wake up" into a dynamic web application in response to specific user actions, is revolutionary. It significantly simplifies the concerns of operating an editing environment with Tina. With this workflow, we don't have to host a separate [Cloud Editing Environment](https://tinacms.org/blog/editing-on-the-cloud) &mdash; **the live site is the editing environment!** And thanks to Git and our fork-based workflow, we can edit content from the production website worry-free.
 
-### How it works
+## Crowdsourcing your content
 
-Before this "preview" feature, page templates in Next.js were either built to be static or used Server-Side Rendering (SSR). Now, the same page template can conditionally take advantage of **both static and SSR data-loading**. This allows us to switch “content” data-sources without rebuilding the site.
+**Visual Open Authoring** is a unique, low-friction approach to website editing. Thanks to the clever Next.js preview mode, Tina's content flexibility, and the seamless [Inline Editing](https://tinacms.org/docs/inline-editing) experience, we're ready for a digital content revolution.
 
-Any time an editor loads a page in "Edit Mode", we can build a unique page for them. This page gets its own data from a particular Git branch or fork. SSR renders the page with the user's dynamic content. We can even allow the user to create new pages that don't exist in the static 'production' build.
+What if it were effortless to fix a spelling error on the New York Times? Imagine contributing to the ReactJS documentation in an instant.
 
-With this workflow, we don't have to host a separate [Cloud Editing Environment](https://tinacms.org/blog/editing-on-the-cloud). The **live site is the editing environment**, with Git to keep the guide rails.
-
-## Crowd-sourcing your content
-
-We see this approach as _lowering the friction_ of website editing as a whole. Combine _Visual Open Authoring_ with [Inline Editing](https://tinacms.org/docs/inline-editing) and we have ourselves a digital-content revolution. What if it were effortless to fix a spelling error on the New York Times? Imagine contributing to the ReactJS documentation in an instant.
-
-**The web is all about community**, crowd-sourcing, and the power of many perspectives. The easier it is to make content contributions, the more viewpoints we incorporate. The less context switching, the _more space for creative expression_ of ideas.
+**The web is all about community**, crowdsourcing, and the power of many perspectives. The easier it is to make content contributions, the more viewpoints we incorporate.
```

Comment:
```suggestion
**The web is all about community**, crowdsourcing, and the power of many perspectives. The easier it is to make content contributions, the more viewpoints we incorporate. The less context switching, the _more space for creative expression_ of ideas.
```

Reply:
I think this line about creativity is important

Reply:
I removed that line for a reason. I'm not quite sure what it means. This conclusion really builds momentum, and that seemed to take it away

Reply:
I think my problem is with its focus on "context switching". This paragraph hits very broad notes. Community, easy contribution, many perspectives, incorporating viewpoints, and.... context-switching?

Why does a lack of context-switching create "space"? What does VOA actually do to foster "space for creative expression"? I'm not denying that creativity is good, but does VOA really make an impact on this specific front?

Reply:
I think the conclusion is trying to paint a bigger vision, with VOA in combination with the broader impact of Tina. Once of the biggest 'differences' with Tina is the aspect of the editing context. And given that it's mentioned in the introduction as what makes this version of OA different, it seems to be an important thread.

Reply:
I guess you're right, the in-context aspect is an important part of the thesis. I'm reasonably convinced, back in it goes